js/gem-notice.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
@fjarrett We can go that route, but I was worried that the notice would get overlooked after the first page load. It is less code that route. If you think that would be best I can adjust the PR accordingly.